src/USER-MISC/pair_agni.cpp

```diff
@@ -187,7 +188,13 @@ void PairAGNI::compute(int eflag, int vflag)
         const double wZ = cF*delz/r;
 
         for (k = 0; k < iparam.numeta; ++k) {
-          const double e = exp(-(iparam.eta[k]*rsq));
+          	double e = 0.0;
```

Comment:
Please retain the LAMMPS code formatting style. Especially:
- use only spaces, no TABs
- use 2 spaces indentation (not 8 or 4)
- no trailing whitespace
- wrap lines of 80 characters and more